<<<<<<< HEAD
error: uncaughtException: listen EADDRINUSE: address already in use :::4000
Error: listen EADDRINUSE: address already in use :::4000
    at Server.setupListenHandle [as _listen2] (node:net:1485:16)
    at listenInCluster (node:net:1533:12)
    at Server.listen (node:net:1621:7)
    at Function.listen (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\application.js:635:24)
    at Object.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\src\/index.js:37:5)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._compile (D:\FundooNotesProject\Fundoo_Notes\node_modules\pirates\lib\index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Object.newLoader [as .js] (D:\FundooNotesProject\Fundoo_Notes\node_modules\pirates\lib\index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12)
    at Function.runMain (node:internal/modules/run_main:81:12)
    at Object.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\node\src\_babel-node.ts:223:12)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {"date":"Tue Jan 31 2023 14:55:09 GMT+0530 (India Standard Time)","error":{"address":"::","code":"EADDRINUSE","errno":-4091,"port":4000,"syscall":"listen"},"exception":true,"os":{"loadavg":[0,0,0],"uptime":16883},"process":{"argv":["node","D:\\FundooNotesProject\\Fundoo_Notes\\src\\index.js"],"cwd":"D:\\FundooNotesProject\\Fundoo_Notes","execPath":"D:\\node\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":19400893,"external":20922902,"heapTotal":92594176,"heapUsed":67627688,"rss":132771840},"pid":15304,"uid":null,"version":"v18.12.0"},"stack":"Error: listen EADDRINUSE: address already in use :::4000\n    at Server.setupListenHandle [as _listen2] (node:net:1485:16)\n    at listenInCluster (node:net:1533:12)\n    at Server.listen (node:net:1621:7)\n    at Function.listen (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\express\\lib\\application.js:635:24)\n    at Object.<anonymous> (D:\\FundooNotesProject\\Fundoo_Notes\\src\\/index.js:37:5)\n    at Module._compile (node:internal/modules/cjs/loader:1159:14)\n    at Module._compile (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\pirates\\lib\\index.js:136:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)\n    at Object.newLoader [as .js] (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\pirates\\lib\\index.js:141:7)\n    at Module.load (node:internal/modules/cjs/loader:1037:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:878:12)\n    at Function.runMain (node:internal/modules/run_main:81:12)\n    at Object.<anonymous> (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\@babel\\node\\src\\_babel-node.ts:223:12)\n    at Module._compile (node:internal/modules/cjs/loader:1159:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1213:10)\n    at Module.load (node:internal/modules/cjs/loader:1037:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:878:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:23:47","timestamp":"2023-01-31T09:25:10.023Z","trace":[{"column":16,"file":"node:net","function":"Server.setupListenHandle [as _listen2]","line":1485,"method":"setupListenHandle [as _listen2]","native":false},{"column":12,"file":"node:net","function":"listenInCluster","line":1533,"method":null,"native":false},{"column":7,"file":"node:net","function":"Server.listen","line":1621,"method":"listen","native":false},{"column":24,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\express\\lib\\application.js","function":"Function.listen","line":635,"method":"listen","native":false},{"column":5,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\src\\/index.js","function":null,"line":37,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1159,"method":"_compile","native":false},{"column":24,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\pirates\\lib\\index.js","function":"Module._compile","line":136,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1213,"method":".js","native":false},{"column":7,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\pirates\\lib\\index.js","function":"Object.newLoader [as .js]","line":141,"method":"js]","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":1037,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":878,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.runMain","line":81,"method":"runMain","native":false},{"column":12,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\@babel\\node\\src\\_babel-node.ts","function":null,"line":223,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1159,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1213,"method":".js","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":1037,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":878,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":81,"method":"executeUserEntryPoint [as runMain]","native":false},{"column":47,"file":"node:internal/main/run_main_module","function":null,"line":23,"method":null,"native":false}]}
=======
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ userId: 'shreya@gmail.com' }" (type Object) at path "_id" for model "Notes"
    at model.Query.exec (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63d9387483ee1531acf7d429 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-31T15:56:27.691Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ userId: 'shreya@gmail.com' }" (type Object) at path "_id" for model "Notes"
    at model.Query.exec (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63d9387483ee1531acf7d429 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-31T15:56:57.793Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "shreya@gmail.com" (type string) at path "_id" for model "Notes"
    at model.Query.exec (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63d9387483ee1531acf7d429 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-31T15:57:28.652Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ userId: 'naruto@gmail.com' }" (type Object) at path "_id" for model "Notes"
    at model.Query.exec (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63d9387483ee1531acf7d429 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-31T16:01:47.887Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ userId: 'naruto@gmail.com' }" (type Object) at path "_id" for model "Notes"
    at model.Query.exec (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63d9387483ee1531acf7d429 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-31T16:02:26.375Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ userId: 'naruto@gmail.com', id: undefined }" (type Object) at path "_id" for model "Notes"
    at model.Query.exec (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63d9387483ee1531acf7d429 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-31T16:03:19.229Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ id: undefined }" (type Object) at path "_id" for model "Notes"
    at model.Query.exec (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63d9387483ee1531acf7d429 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-31T16:03:59.773Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "63d9387483ee1531acf7d429k" (type string) at path "_id" for model "Notes"
    at model.Query.exec (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63d9387483ee1531acf7d429k 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-31T16:13:22.274Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "63d9387483ee1531acf7d42" (type string) at path "_id" for model "Notes"
    at model.Query.exec (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63d9387483ee1531acf7d42 
    method - GET 
    IP - ::1
   {"timestamp":"2023-01-31T16:13:33.557Z"}
>>>>>>> notes
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: 'shreya@gmail.com', userId: '63d8ff42a0deeb15a4266ea7' }" (type Object) at path "_id" for model "Notes"
    at model.Query.exec (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\FundooNotesProject\Fundoo_Notes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/63d8ff42a0deeb15a4266ea7 
    method - PUT 
    IP - ::1
   {"timestamp":"2023-02-01T06:15:30.375Z"}
error: 
    status - 500 
    message - Error: data and salt arguments required
    at Object.hash (D:\FundooNotesProject\Fundoo_Notes\node_modules\bcrypt\bcrypt.js:137:17)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\bcrypt\promises.js:29:12
    at new Promise (<anonymous>)
    at Object.module.exports.promise (D:\FundooNotesProject\Fundoo_Notes\node_modules\bcrypt\promises.js:20:12)
    at Object.hash (D:\FundooNotesProject\Fundoo_Notes\node_modules\bcrypt\bcrypt.js:133:25)
    at _callee4$ (D:\FundooNotesProject\Fundoo_Notes\src\services\/user.service.js:64:32)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.pwdReset (D:\FundooNotesProject\Fundoo_Notes\src\services\/user.service.js:61:22)
    at _callee4$ (D:\FundooNotesProject\Fundoo_Notes\src\controllers\/user.controller.js:54:36)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at pwdReset (D:\FundooNotesProject\Fundoo_Notes\src\controllers\/user.controller.js:51:22)
    at Layer.handle [as handle_request] (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at _callee2$ (D:\FundooNotesProject\Fundoo_Notes\src\middlewares\/auth.middleware.js:43:5)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/reset/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6InNhcHRhcnNoaTE5OTliaXN3YXNAZ21haWwuY29tIiwiaWF0IjoxNjc1Njk5NDI1fQ.OdkWOgItRe3F_t4A2nuJLkK1EVb_GW9q1Mq6eykDmuE 
    method - PUT 
    IP - ::1
   {"timestamp":"2023-02-06T16:04:29.150Z"}
error: uncaughtException: expected 404 to equal 201
AssertionError: expected 404 to equal 201
    at Test.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\tests\integration\/user.test.js:42:40)
    at Test.assert (D:\FundooNotesProject\Fundoo_Notes\node_modules\supertest\lib\test.js:181:6)
    at Server.localAssert (D:\FundooNotesProject\Fundoo_Notes\node_modules\supertest\lib\test.js:131:12)
    at Object.onceWrapper (node:events:627:28)
    at Server.emit (node:events:513:28)
    at emitCloseNT (node:net:1880:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Mon Feb 06 2023 21:55:35 GMT+0530 (India Standard Time)","error":{"actual":404,"expected":201,"message":"expected 404 to equal 201","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 404 to equal 201\n    at Test.<anonymous> (D:\\FundooNotesProject\\Fundoo_Notes\\tests\\integration\\/user.test.js:42:40)\n    at Test.assert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:181:6)\n    at Server.localAssert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:131:12)\n    at Object.onceWrapper (node:events:627:28)\n    at Server.emit (node:events:513:28)\n    at emitCloseNT (node:net:1880:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[0,0,0],"uptime":3834},"process":{"argv":["D:\\node\\node.exe","D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\mocha\\bin\\mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/**/*.js"],"cwd":"D:\\FundooNotesProject\\Fundoo_Notes","execPath":"D:\\node\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":19103833,"external":20232467,"heapTotal":135434240,"heapUsed":112190624,"rss":174325760},"pid":14364,"uid":null,"version":"v18.12.0"},"stack":"AssertionError: expected 404 to equal 201\n    at Test.<anonymous> (D:\\FundooNotesProject\\Fundoo_Notes\\tests\\integration\\/user.test.js:42:40)\n    at Test.assert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:181:6)\n    at Server.localAssert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:131:12)\n    at Object.onceWrapper (node:events:627:28)\n    at Server.emit (node:events:513:28)\n    at emitCloseNT (node:net:1880:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-02-06T16:25:35.399Z","trace":[{"column":40,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\tests\\integration\\/user.test.js","function":null,"line":42,"method":null,"native":false},{"column":6,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":627,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":513,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":1880,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: uncaughtException: expected 404 to equal 201
AssertionError: expected 404 to equal 201
    at Test.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\tests\integration\/user.test.js:42:40)
    at Test.assert (D:\FundooNotesProject\Fundoo_Notes\node_modules\supertest\lib\test.js:181:6)
    at Server.localAssert (D:\FundooNotesProject\Fundoo_Notes\node_modules\supertest\lib\test.js:131:12)
    at Object.onceWrapper (node:events:627:28)
    at Server.emit (node:events:513:28)
    at emitCloseNT (node:net:1880:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Mon Feb 06 2023 21:56:51 GMT+0530 (India Standard Time)","error":{"actual":404,"expected":201,"message":"expected 404 to equal 201","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 404 to equal 201\n    at Test.<anonymous> (D:\\FundooNotesProject\\Fundoo_Notes\\tests\\integration\\/user.test.js:42:40)\n    at Test.assert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:181:6)\n    at Server.localAssert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:131:12)\n    at Object.onceWrapper (node:events:627:28)\n    at Server.emit (node:events:513:28)\n    at emitCloseNT (node:net:1880:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[0,0,0],"uptime":3911},"process":{"argv":["D:\\node\\node.exe","D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\mocha\\bin\\mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/**/*.js"],"cwd":"D:\\FundooNotesProject\\Fundoo_Notes","execPath":"D:\\node\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":19371598,"external":20495997,"heapTotal":125071360,"heapUsed":98840232,"rss":162078720},"pid":13132,"uid":null,"version":"v18.12.0"},"stack":"AssertionError: expected 404 to equal 201\n    at Test.<anonymous> (D:\\FundooNotesProject\\Fundoo_Notes\\tests\\integration\\/user.test.js:42:40)\n    at Test.assert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:181:6)\n    at Server.localAssert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:131:12)\n    at Object.onceWrapper (node:events:627:28)\n    at Server.emit (node:events:513:28)\n    at emitCloseNT (node:net:1880:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-02-06T16:26:51.702Z","trace":[{"column":40,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\tests\\integration\\/user.test.js","function":null,"line":42,"method":null,"native":false},{"column":6,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":627,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":513,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":1880,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: uncaughtException: expected 404 to equal 201
AssertionError: expected 404 to equal 201
    at Test.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\tests\integration\/user.test.js:42:40)
    at Test.assert (D:\FundooNotesProject\Fundoo_Notes\node_modules\supertest\lib\test.js:181:6)
    at Server.localAssert (D:\FundooNotesProject\Fundoo_Notes\node_modules\supertest\lib\test.js:131:12)
    at Object.onceWrapper (node:events:627:28)
    at Server.emit (node:events:513:28)
    at emitCloseNT (node:net:1880:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Mon Feb 06 2023 21:58:54 GMT+0530 (India Standard Time)","error":{"actual":404,"expected":201,"message":"expected 404 to equal 201","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 404 to equal 201\n    at Test.<anonymous> (D:\\FundooNotesProject\\Fundoo_Notes\\tests\\integration\\/user.test.js:42:40)\n    at Test.assert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:181:6)\n    at Server.localAssert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:131:12)\n    at Object.onceWrapper (node:events:627:28)\n    at Server.emit (node:events:513:28)\n    at emitCloseNT (node:net:1880:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[0,0,0],"uptime":4034},"process":{"argv":["D:\\node\\node.exe","D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\mocha\\bin\\mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/**/*.js"],"cwd":"D:\\FundooNotesProject\\Fundoo_Notes","execPath":"D:\\node\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":19443801,"external":20568693,"heapTotal":124284928,"heapUsed":98361280,"rss":161071104},"pid":8276,"uid":null,"version":"v18.12.0"},"stack":"AssertionError: expected 404 to equal 201\n    at Test.<anonymous> (D:\\FundooNotesProject\\Fundoo_Notes\\tests\\integration\\/user.test.js:42:40)\n    at Test.assert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:181:6)\n    at Server.localAssert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:131:12)\n    at Object.onceWrapper (node:events:627:28)\n    at Server.emit (node:events:513:28)\n    at emitCloseNT (node:net:1880:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-02-06T16:28:54.681Z","trace":[{"column":40,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\tests\\integration\\/user.test.js","function":null,"line":42,"method":null,"native":false},{"column":6,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":627,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":513,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":1880,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: uncaughtException: expected 404 to equal 201
AssertionError: expected 404 to equal 201
    at Test.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\tests\integration\/user.test.js:42:40)
    at Test.assert (D:\FundooNotesProject\Fundoo_Notes\node_modules\supertest\lib\test.js:181:6)
    at Server.localAssert (D:\FundooNotesProject\Fundoo_Notes\node_modules\supertest\lib\test.js:131:12)
    at Object.onceWrapper (node:events:627:28)
    at Server.emit (node:events:513:28)
    at emitCloseNT (node:net:1880:8)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {"date":"Mon Feb 06 2023 21:59:46 GMT+0530 (India Standard Time)","error":{"actual":404,"expected":201,"message":"expected 404 to equal 201","name":"AssertionError","operator":"strictEqual","showDiff":true,"stack":"AssertionError: expected 404 to equal 201\n    at Test.<anonymous> (D:\\FundooNotesProject\\Fundoo_Notes\\tests\\integration\\/user.test.js:42:40)\n    at Test.assert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:181:6)\n    at Server.localAssert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:131:12)\n    at Object.onceWrapper (node:events:627:28)\n    at Server.emit (node:events:513:28)\n    at emitCloseNT (node:net:1880:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)"},"exception":true,"os":{"loadavg":[0,0,0],"uptime":4085},"process":{"argv":["D:\\node\\node.exe","D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\mocha\\bin\\mocha","--exit","--recursive","--require","@babel/register","--timeout=5000","tests/**/*.js"],"cwd":"D:\\FundooNotesProject\\Fundoo_Notes","execPath":"D:\\node\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":19200829,"external":20325141,"heapTotal":120442880,"heapUsed":95018664,"rss":157597696},"pid":8756,"uid":null,"version":"v18.12.0"},"stack":"AssertionError: expected 404 to equal 201\n    at Test.<anonymous> (D:\\FundooNotesProject\\Fundoo_Notes\\tests\\integration\\/user.test.js:42:40)\n    at Test.assert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:181:6)\n    at Server.localAssert (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js:131:12)\n    at Object.onceWrapper (node:events:627:28)\n    at Server.emit (node:events:513:28)\n    at emitCloseNT (node:net:1880:8)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","timestamp":"2023-02-06T16:29:46.388Z","trace":[{"column":40,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\tests\\integration\\/user.test.js","function":null,"line":42,"method":null,"native":false},{"column":6,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js","function":"Test.assert","line":181,"method":"assert","native":false},{"column":12,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\supertest\\lib\\test.js","function":"Server.localAssert","line":131,"method":"localAssert","native":false},{"column":28,"file":"node:events","function":"Object.onceWrapper","line":627,"method":"onceWrapper","native":false},{"column":28,"file":"node:events","function":"Server.emit","line":513,"method":"emit","native":false},{"column":8,"file":"node:net","function":"emitCloseNT","line":1880,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
error: 
    status - 500 
    message - ValidationError: "fname" length must be at least 4 characters long 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:32:33.806Z"}
error: 
    status - 500 
    message - ValidationError: "lname" length must be at least 4 characters long 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:32:33.827Z"}
error: 
      status - 11000
      message - E11000 duplicate key error collection: test.users index: Email_1 dup key: { Email: null } 
      url - /api/v1/users/ 
      method - POST 
      IP - ::ffff:127.0.0.1
     {"timestamp":"2023-02-06T16:32:38.443Z"}
error: 
    status - 500 
    message - RangeError: Invalid status code: 11000
    at new NodeError (node:internal/errors:393:5)
    at ServerResponse.writeHead (node:_http_server:343:11)
    at ServerResponse.writeHead (D:\FundooNotesProject\Fundoo_Notes\node_modules\on-headers\index.js:44:26)
    at ServerResponse._implicitHeader (node:_http_server:334:8)
    at write_ (node:_http_outgoing:867:9)
    at ServerResponse.end (node:_http_outgoing:977:5)
    at ServerResponse.send (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\response.js:232:10)
    at ServerResponse.json (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\response.js:278:15)
    at appErrorHandler (D:\FundooNotesProject\Fundoo_Notes\src\middlewares\/error.middleware.js:36:26)
    at Layer.handle_error (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:326:13)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:646:15
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:265:14)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:646:15
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:265:14)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\route.js:136:14)
    at _callee$ (D:\FundooNotesProject\Fundoo_Notes\src\controllers\/user.controller.js:14:5)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.throw (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _throw (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:32:38.450Z"}
error: 
    status - 500 
    message - ValidationError: "password" must be a string 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:32:38.462Z"}
error: 
    status - 500 
    message - Error: Error: Invalid Email
    at _callee2$ (D:\FundooNotesProject\Fundoo_Notes\src\services\/user.service.js:38:11)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:32:39.058Z"}
error: 
    status - 500 
    message - Error: Error: Invalid Email
    at _callee2$ (D:\FundooNotesProject\Fundoo_Notes\src\services\/user.service.js:38:11)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:32:39.309Z"}
error: 
    status - 500 
    message - ValidationError: "fname" length must be at least 4 characters long 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:33:50.357Z"}
error: 
    status - 500 
    message - ValidationError: "lname" length must be at least 4 characters long 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:33:50.365Z"}
error: 
      status - 11000
      message - E11000 duplicate key error collection: test.users index: Email_1 dup key: { Email: null } 
      url - /api/v1/users/ 
      method - POST 
      IP - ::ffff:127.0.0.1
     {"timestamp":"2023-02-06T16:33:50.919Z"}
error: 
    status - 500 
    message - RangeError: Invalid status code: 11000
    at new NodeError (node:internal/errors:393:5)
    at ServerResponse.writeHead (node:_http_server:343:11)
    at ServerResponse.writeHead (D:\FundooNotesProject\Fundoo_Notes\node_modules\on-headers\index.js:44:26)
    at ServerResponse._implicitHeader (node:_http_server:334:8)
    at write_ (node:_http_outgoing:867:9)
    at ServerResponse.end (node:_http_outgoing:977:5)
    at ServerResponse.send (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\response.js:232:10)
    at ServerResponse.json (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\response.js:278:15)
    at appErrorHandler (D:\FundooNotesProject\Fundoo_Notes\src\middlewares\/error.middleware.js:36:26)
    at Layer.handle_error (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:326:13)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:646:15
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:265:14)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:646:15
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:265:14)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\route.js:136:14)
    at _callee$ (D:\FundooNotesProject\Fundoo_Notes\src\controllers\/user.controller.js:14:5)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.throw (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _throw (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:33:50.926Z"}
error: 
    status - 500 
    message - ValidationError: "password" must be a string 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:33:50.937Z"}
error: 
    status - 500 
    message - Error: Error: Invalid Email
    at _callee2$ (D:\FundooNotesProject\Fundoo_Notes\src\services\/user.service.js:38:11)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:33:51.511Z"}
error: 
    status - 500 
    message - Error: Error: Invalid Email
    at _callee2$ (D:\FundooNotesProject\Fundoo_Notes\src\services\/user.service.js:38:11)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:33:51.781Z"}
error: 
    status - 500 
    message - ValidationError: "fname" length must be at least 4 characters long 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:35:30.618Z"}
error: 
    status - 500 
    message - ValidationError: "lname" length must be at least 4 characters long 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:35:30.625Z"}
error: 
      status - 11000
      message - E11000 duplicate key error collection: test.users index: Email_1 dup key: { Email: null } 
      url - /api/v1/users/ 
      method - POST 
      IP - ::ffff:127.0.0.1
     {"timestamp":"2023-02-06T16:35:31.169Z"}
error: 
    status - 500 
    message - RangeError: Invalid status code: 11000
    at new NodeError (node:internal/errors:393:5)
    at ServerResponse.writeHead (node:_http_server:343:11)
    at ServerResponse.writeHead (D:\FundooNotesProject\Fundoo_Notes\node_modules\on-headers\index.js:44:26)
    at ServerResponse._implicitHeader (node:_http_server:334:8)
    at write_ (node:_http_outgoing:867:9)
    at ServerResponse.end (node:_http_outgoing:977:5)
    at ServerResponse.send (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\response.js:232:10)
    at ServerResponse.json (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\response.js:278:15)
    at appErrorHandler (D:\FundooNotesProject\Fundoo_Notes\src\middlewares\/error.middleware.js:36:26)
    at Layer.handle_error (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:326:13)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:646:15
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:265:14)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:646:15
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:265:14)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\route.js:136:14)
    at _callee$ (D:\FundooNotesProject\Fundoo_Notes\src\controllers\/user.controller.js:14:5)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.throw (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _throw (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:35:31.177Z"}
error: 
    status - 500 
    message - ValidationError: "password" must be a string 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:35:31.194Z"}
error: 
    status - 500 
    message - Error: Error: Invalid Email
    at _callee2$ (D:\FundooNotesProject\Fundoo_Notes\src\services\/user.service.js:38:11)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:35:31.812Z"}
error: 
    status - 500 
    message - Error: Error: Invalid Email
    at _callee2$ (D:\FundooNotesProject\Fundoo_Notes\src\services\/user.service.js:38:11)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T16:35:32.095Z"}
error: 
    status - 500 
    message - ReferenceError: bearertToken is not defined
    at _callee2$ (D:\FundooNotesProject\Fundoo_Notes\src\middlewares\/auth.middleware.js:40:35)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at resetAuth (D:\FundooNotesProject\Fundoo_Notes\src\middlewares\/auth.middleware.js:30:23)
    at Layer.handle [as handle_request] (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at param (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:365:14)
    at param (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:421:3)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\FundooNotesProject\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (D:\FundooNotesProject\Fundoo_Notes\node_modules\raw-body\index.js:231:16)
    at done (D:\FundooNotesProject\Fundoo_Notes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\FundooNotesProject\Fundoo_Notes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:513:28)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1359:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/users/resetPassword/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6InNhcHRhcnNoaTE5OTliaXN3YXNAZ21haWwuY29tIiwiaWF0IjoxNjc1Njk5NDI1fQ.OdkWOgItRe3F_t4A2nuJLkK1EVb_GW9q1Mq6eykDmuE 
    method - PUT 
    IP - ::1
   {"timestamp":"2023-02-06T16:54:46.652Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\FundooNotesProject\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\FundooNotesProject\Fundoo_Notes\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee2$ (D:\FundooNotesProject\Fundoo_Notes\src\middlewares\/auth.middleware.js:40:28)
    at tryCatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at resetAuth (D:\FundooNotesProject\Fundoo_Notes\src\middlewares\/auth.middleware.js:30:23)
    at Layer.handle [as handle_request] (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at param (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:365:14)
    at param (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:421:3)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\FundooNotesProject\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at D:\FundooNotesProject\Fundoo_Notes\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (D:\FundooNotesProject\Fundoo_Notes\node_modules\raw-body\index.js:231:16)
    at done (D:\FundooNotesProject\Fundoo_Notes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\FundooNotesProject\Fundoo_Notes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:513:28)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1359:12) 
    url - /api/v1/users/resetPassword/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6InNhcHRhcnNoaTE5OTliaXN3YXNAZ21haWwuY29tIiwiaWF0IjoxNjc1Njk5NDI1fQ.OdkWOgItRe3F_t4A2nuJLkK1EVb_GW9q1Mq6eykDmuE 
    method - PUT 
    IP - ::1
   {"timestamp":"2023-02-06T16:55:39.466Z"}
error: uncaughtException: listen EADDRINUSE: address already in use :::3000
Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1485:16)
    at listenInCluster (node:net:1533:12)
    at Server.listen (node:net:1621:7)
    at Function.listen (D:\FundooNotesProject\Fundoo_Notes\node_modules\express\lib\application.js:635:24)
    at Object.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\src\/index.js:37:5)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._compile (D:\FundooNotesProject\Fundoo_Notes\node_modules\pirates\lib\index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Object.newLoader [as .js] (D:\FundooNotesProject\Fundoo_Notes\node_modules\pirates\lib\index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12)
    at Function.runMain (node:internal/modules/run_main:81:12)
    at Object.<anonymous> (D:\FundooNotesProject\Fundoo_Notes\node_modules\@babel\node\src\_babel-node.ts:223:12)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32) {"date":"Mon Feb 06 2023 22:38:37 GMT+0530 (India Standard Time)","error":{"address":"::","code":"EADDRINUSE","errno":-4091,"port":3000,"syscall":"listen"},"exception":true,"os":{"loadavg":[0,0,0],"uptime":6417},"process":{"argv":["node","D:\\FundooNotesProject\\Fundoo_Notes\\src\\index.js"],"cwd":"D:\\FundooNotesProject\\Fundoo_Notes","execPath":"D:\\node\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18467246,"external":20168110,"heapTotal":121106432,"heapUsed":88634376,"rss":161341440},"pid":15208,"uid":null,"version":"v18.12.0"},"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1485:16)\n    at listenInCluster (node:net:1533:12)\n    at Server.listen (node:net:1621:7)\n    at Function.listen (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\express\\lib\\application.js:635:24)\n    at Object.<anonymous> (D:\\FundooNotesProject\\Fundoo_Notes\\src\\/index.js:37:5)\n    at Module._compile (node:internal/modules/cjs/loader:1159:14)\n    at Module._compile (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\pirates\\lib\\index.js:136:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)\n    at Object.newLoader [as .js] (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\pirates\\lib\\index.js:141:7)\n    at Module.load (node:internal/modules/cjs/loader:1037:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:878:12)\n    at Function.runMain (node:internal/modules/run_main:81:12)\n    at Object.<anonymous> (D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\@babel\\node\\src\\_babel-node.ts:223:12)\n    at Module._compile (node:internal/modules/cjs/loader:1159:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1213:10)\n    at Module.load (node:internal/modules/cjs/loader:1037:32)","timestamp":"2023-02-06T17:08:37.631Z","trace":[{"column":16,"file":"node:net","function":"Server.setupListenHandle [as _listen2]","line":1485,"method":"setupListenHandle [as _listen2]","native":false},{"column":12,"file":"node:net","function":"listenInCluster","line":1533,"method":null,"native":false},{"column":7,"file":"node:net","function":"Server.listen","line":1621,"method":"listen","native":false},{"column":24,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\express\\lib\\application.js","function":"Function.listen","line":635,"method":"listen","native":false},{"column":5,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\src\\/index.js","function":null,"line":37,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1159,"method":"_compile","native":false},{"column":24,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\pirates\\lib\\index.js","function":"Module._compile","line":136,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1213,"method":".js","native":false},{"column":7,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\pirates\\lib\\index.js","function":"Object.newLoader [as .js]","line":141,"method":"js]","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":1037,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":878,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.runMain","line":81,"method":"runMain","native":false},{"column":12,"file":"D:\\FundooNotesProject\\Fundoo_Notes\\node_modules\\@babel\\node\\src\\_babel-node.ts","function":null,"line":223,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1159,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1213,"method":".js","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":1037,"method":"load","native":false}]}
error: 
    status - 500 
    message - ValidationError: "fname" length must be at least 4 characters long 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T17:30:21.508Z"}
error: 
    status - 500 
    message - ValidationError: "lname" length must be at least 4 characters long 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T17:30:21.514Z"}
error: 
    status - 500 
    message - ValidationError: "email" must be a valid email 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T17:30:21.522Z"}
error: 
    status - 500 
    message - ValidationError: "password" must be a string 
    url - /api/v1/users/ 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2023-02-06T17:30:21.530Z"}
